internal/oci/runtime_vm.go

```diff
@@ -337,13 +337,22 @@ func (r *runtimeVM) execContainerCommon(c *Container, cmd []string, timeout int6
 	}
 	defer execIO.Close()
 
+	// chan to notify that can call runtime's CloseIO API
+	closeIOChan := make(chan bool)
+	defer func() {
+		if closeIOChan != nil {
+			close(closeIOChan)
+		}
+	}()
+
 	execIO.Attach(cio.AttachOptions{
 		Stdin:     stdin,
 		Stdout:    stdout,
 		Stderr:    stderr,
 		Tty:       tty,
 		StdinOnce: true,
 		CloseStdin: func() error {
+			<-closeIOChan
```

Comment:
Considering that's the case, this piece wouldn't trigger closeIO twice?

If so, would be okay to just close the channel and **not** call r.closeIO() here?

Reply:
As above explained, `CloseStdin` will be blocked to call the real underlay `r.closeIO()`